docker-alfresco/test/docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
                 -Dftp.enabled=true
```

Comment:
PR title says: REPO-3256: EOL:Disable CIFS and FTP by default

but the file change still shows:

-Dftp.enabled=true

?

Reply:
/test/docker-compose.yml is used for running TAS tests and it is needed for integration tests